services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfeLinkBolt.java

```diff
@@ -441,6 +442,9 @@ protected void dispatchMain(Tuple tuple, InfoMessage infoMessage) {
             handleIslEvent(tuple, infoMessage);
         } else if (data instanceof DiscoPacketSendingConfirmation) {
             handleSentDiscoPacket((DiscoPacketSendingConfirmation) data);
+        } else if (data instanceof DeactivateIslInfoData) {
+            DeactivateIslInfoData deactivateIslInfoData = (DeactivateIslInfoData) data;
+            discovery.handleFailed(deactivateIslInfoData.getSrcSwitchId(), deactivateIslInfoData.getSrcPort());
```

Comment:
what if we are in the middle of sync process?

Reply:
Good point! Now this event is handled before the condition for determining the state of the bolt. But still need to check the behavior of this code during synchronization.